ansible_base/lib/utils/encryption.py

```diff
@@ -65,9 +65,8 @@ def decrypt_string(self, value: str) -> str:
 
         # Finally decode the value
         encrypted = base64.b64decode(b64data)
-        value = self.decrypt(encrypted)
```

Comment:
It doesn't like it when you re-define an input argument with a type that doesn't match its annotation. Fair, I say.

Reply:
Thats interesting, good nugget.